Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM alpine:latest
+FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
 
 WORKDIR /
 
-RUN apk add -U --no-cache ca-certificates xfsprogs xfsprogs-extra
-
 COPY direct-csi /direct-csi
 COPY CREDITS /licenses/CREDITS
 COPY LICENSE /licenses/LICENSE
 
-ENTRYPOINT ["/direct-csi"]
+COPY centos7.repo /etc/yum.repos.d/centos7.repo
+COPY RPM-GPG-KEY-CentOS-7 /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
```

Comment:
Instead of COPY-ing, we can actually use curl and redirect it to `/etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7`